qt/scientific_interfaces/Indirect/ISISEnergyTransfer.cpp

```diff
@@ -516,8 +527,13 @@ void ISISEnergyTransfer::algorithmComplete(bool error) {
       m_outputWorkspaces = outputGroup->getNames();
       m_pythonExportWsName = m_outputWorkspaces[0];
 
-      if (!m_uiForm.ckGroupOutput->isChecked())
+      if (m_uiForm.cbGroupOutput->currentText() == "Ungrouped") {
         ungroupWorkspace(outputGroup->getName());
+      } else if (m_uiForm.cbGroupOutput->currentText() == "Group by Sample") {
```

Comment:
Is there a way to track this by index / enum / something else?
Just because this is quite brittle to a future developer ever changing `Ungrouped` or `Group by Sample` to something like `Group By Sample` which would break it